<launch>
  <arg name="lanelet2_map_topic" default="vector_map"/>
  <arg name="lanelet2_map_marker_topic" default="vector_map_marker"/>

  <node pkg="autoware_lanelet2_map_visualizer" exec="lanelet2_map_visualization" name="lanelet2_map_visualization" output="both">
    <remap from="input/lanelet2_map" to="$(var lanelet2_map_topic)"/>
    <remap from="output/lanelet2_map_marker" to="$(var lanelet2_map_marker_topic)"/>
  </node>
</launch>
